### PR TITLE
Adds logs for the order flow

### PR DESCRIPTION
**Pull request by ftbits**
Adds logs for the order flow.

### CODE REVIEW

File: Plantix/buyer-app/flow-2/confirm.json [hunk 1]
```diff
@@ -0,0 +1,173 @@
+{
+    "context": {
+        "action": "confirm",
+        "bap_id": "onca.dev.peat-cloud",
+        "bap_uri": "https://newest-plants-phillips-probably.trycloudflare.com/ondc/v1",
```
Comment (76tr75b) [inline Plantix/buyer-app/flow-2/confirm.json]:
```suggestion
        "bap_id": "onca.dev.peat-cloud.com",
        "bap_uri": "https://onca.dev.peat-cloud.com/ondc/v1",
```

File: Plantix/buyer-app/flow-2/select.json [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+                },
+                {
+                    "id": "4vkgqsSvRZEwywbpMeW2",
+                    "location_id": "7kZu3N7P57bFinuL1yha",
+                    "quantity": {
+                        "count": 4
+                    }
+                }
+            ],
+            "offers": null,
```
Comment (76tr75b) [inline Plantix/buyer-app/flow-2/select.json]: Maybe we should use `exclude_nones=True` when generating requests.

File: Plantix/buyer-app/flow-2/on_select.json [hunk 3]
```diff
@@ -0,0 +1,154 @@
...
+                    },
+                    {
+                        "@ondc/org/item_id": "43d79997-ac7d-4e0c-9b44-deca0e1a2da5",
+                        "title": "Delivery charges",
+                        "@ondc/org/title_type": "delivery",
+                        "price": {
+                            "currency": "INR",
+                            "value": "50"
+                        }
+                    }
```
Comment (76tr75b) [inline Plantix/buyer-app/flow-2/on_select.json:148]: Where is `misc`/buyer app finders fee?

File: Plantix/buyer-app/flow-2/confirm.json [hunk 4]
```diff
@@ -0,0 +1,173 @@
...
+                    "currency": "INR",
+                    "transaction_id": ""
+                },
+                "status": "NOT-PAID",
+                "type": "ON-ORDER",
+                "collected_by": "BAP",
+                "buyer_app_finder_fee_type": null,
+                "buyer_app_finder_fee_amount": null,
+                "settlement_details": null
+            },
```
Comment (76tr75b) [inline Plantix/buyer-app/flow-2/confirm.json]: I think if `"type": "ON-ORDER"` what we also want we need to have `"status": "PAID"` here.
And of course finders fee is missing here.

File: Plantix/buyer-app/flow-2/select.json (hunk 2 above)
Comment (ftbits) [reply Plantix/buyer-app/flow-2/select.json]: I'll add this, and manually remove the `null`s.

File: Plantix/buyer-app/flow-2/on_select.json (hunk 3 above)
Comment (ftbits) [reply Plantix/buyer-app/flow-2/on_select.json:148]: Apparently, we did not receive it.

File: Plantix/buyer-app/flow-4/on_cancel.json [hunk 5]
```diff
@@ -0,0 +1,319 @@
...
+                    "area_code": "123456"
+                },
+                "tax_number": "TAX123456",
+                "email": "john.doe@example.com",
+                "phone": "7702863154",
+                "created_at": "2024-11-07T09:55:03.351149+00:00",
+                "updated_at": "2024-11-07T09:55:03.351278+00:00"
+            },
+            "cancellation": {
+                "cancelled_by": "onca.dev.peat-cloud",
```
Comment (76tr75b) [inline Plantix/buyer-app/flow-4/on_cancel.json:91]:
```suggestion
                "cancelled_by": "onca.dev.peat-cloud.com",
```

File: Plantix/buyer-app/flow-4/on_cancel.json [hunk 6]
```diff
@@ -0,0 +1,319 @@
...
+                        {
+                            "code": "cancel_request",
+                            "list": [
+                                {
+                                    "code": "reason_id",
+                                    "value": "006"
+                                },
+                                {
+                                    "code": "initiated_by",
+                                    "value": "onca.dev.peat-cloud"
```
Comment (76tr75b) [inline Plantix/buyer-app/flow-4/on_cancel.json:197]:
```suggestion
                                    "value": "onca.dev.peat-cloud.com"
```

File: Plantix/buyer-app/flow-4/on_status-after-cancel.json [hunk 7]
```diff
@@ -0,0 +1,205 @@
...
+                    "transaction_id": ""
+                },
+                "status": "PAID",
+                "type": "ON-ORDER",
+                "collected_by": "BAP"
+            },
+            "created_at": "2024-11-07T09:56:37.440462+00:00",
+            "updated_at": "2024-11-07T10:42:05.859Z",
+            "cancellation": {
+                "cancelled_by": "onca.dev.peat-cloud",
```
Comment (76tr75b) [inline Plantix/buyer-app/flow-4/on_status-after-cancel.json:198]:
```suggestion
                "cancelled_by": "onca.dev.peat-cloud.com",
```

File: Plantix/buyer-app/flow-2/confirm.json [hunk 8]
```diff
@@ -0,0 +1,170 @@
...
+                            "value": "300"
+                        },
+                        "item": {
+                            "currency": "INR",
+                            "value": "300"
+                        }
+                    },
+                    {
+                        "item_id": "43d79997-ac7d-4e0c-9b44-deca0e1a2da5",
+                        "item_quantity": null,
```
Comment (76tr75b) [inline Plantix/buyer-app/flow-2/confirm.json:137]: Should we remove this?

File: Plantix/buyer-app/flow-2/confirm.json [hunk 9]
```diff
@@ -0,0 +1,170 @@
...
+                "status": "PAID",
+                "type": "ON-ORDER",
+                "collected_by": "BAP"
+            },
+            "quote": {
+                "price": {
+                    "currency": "INR",
+                    "value": "2150.00"
+                },
+                "breakup": [
```
Comment (76tr75b) [inline Plantix/buyer-app/flow-2/confirm.json:86]: Is it possible that we need to add the misc part by our self in any step in the order flow?

**Review by 76tr75b — COMMENTED** [review]
I did only take a short look because it would take for ever to review everything and i would have to look up each field in the api docs

**Pull request merged**